### ISSUE TITLE
Messages from people are repeated back to them

### ISSUE THREAD

**Issue by joizjpodsjfposajisa** [tijder/SmsMatrix#51]
Installed the app from fdroid and set it up. 2 people sent me messages which did indeed create rooms in Matrix. However, each message they sent me was replied back to them via SMS - got very annoying very quickly! This is in Lineage if it helps.

**Comment by tijder** [comment]
Did you maybe wrote the botname at the place where the username is suppose to be?

**Comment by xpseudonym** [comment]
Yes, I can perhaps confirm this. Re the username/ botname question - does the attached screenshot show correct entries? Thx
![Screenshot_20201107-115921](https://user-images.githubusercontent.com/5032475/98440521-10714d00-20f1-11eb-96a4-7b552ac2ef49.png)

**Comment by charims** [comment]
Necroreply, sorry:

"Bot Username" should be a new user created specifically to be the bot.
"Your username" should be the user you login to matrix with.

**Comment by nachadan** [comment]
I have the same problem, when i write login to matrix to both it works, but it sending messages but also it sends sender message back to the sender. How can i create a new bot user please? Thank you very much
